Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -2018,6 +2018,8 @@
     <string name="incompatible_advertising_not_supported">"Your smartphone can only receive COVID-19 notifications via Bluetooth, but not send them. This means you can be warned of exposures through this interface, but you cannot warn others of exposures yourself. You can send and receive warnings that result from check-ins."</string>
```

Comment:
1. Is it too late to suggest a change to the app message in English?
"Your smartphone can only receive COVID-19 notifications via Bluetooth, but not send them."
"Your smartphone cannot send or receive COVID-19 notifications via Bluetooth."

and use exactly the same term as the Google UI uses, i.e. "COVID-19 Exposure Notifications".

The German text is fine, since it uses the term "COVID-19-Benachrichtigungen" exactly like Google does.

Reply:
Sorry for late respone – since UA already approved this one I'd suggest to open a new PR / issue about this

Reply:
@fynngodau 
Perhaps we could get a comment here from @ChristinaTsigganos before opening any new issue / PR?